### PR TITLE
Version1.0

### PR DESCRIPTION
**Pull request by vladpetrovskyi**

### CODE REVIEW

File: sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,6 @@
+sonar.organization=<vladpetrovskyi>
+sonar.projectKey=<vladpetrovskyi_project_internet-shop>
+
+# relative paths to source directories. More details and properties are described
+# in https://sonarcloud.io/documentation/project-administration/narrowing-the-focus/
+sonar.sources=.
```
Comment (dmytro-hryhoruk) [inline sonar-project.properties:6]: empty line

File: src/main/java/com/internetshop/dao/jdbc/UserDaoJdbcImpl.java [hunk 2]
```diff
@@ -8,45 +14,35 @@
...
-import vlad.petrovskyi.internetshop.exceptions.DataProcessingException;
-import vlad.petrovskyi.internetshop.lib.Dao;
-import vlad.petrovskyi.internetshop.model.Role;
-import vlad.petrovskyi.internetshop.model.User;
-import vlad.petrovskyi.internetshop.util.ConnectionUtil;
 
 @Dao
 public class UserDaoJdbcImpl implements UserDao {
 
+    private static final String ACTION_1 = "SELECT * FROM users JOIN users_roles "
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/internetshop/dao/jdbc/UserDaoJdbcImpl.java:21]:
```suggestion
    private static final String SELECT_USER_BY_PARAMETER = "SELECT * FROM users JOIN users_roles "
```

File: system.properties [hunk 3]
```diff
@@ -0,0 +1 @@
+java.runtime.version=14
```
Comment (dmytro-hryhoruk) [inline system.properties:1]: line

File: Procfile [hunk 4]
```diff
@@ -0,0 +1 @@
+web: java $JAVA_OPTS -jar target/dependency/webapp-runner.jar --port $PORT target/*.war
```
[the remark this answers is not in the document]
Comment (vladpetrovskyi) [reply Procfile:1]: this one is needed to deploy the app to Heroku web-service

**Pull request merged**